pkg/settings/cresettings/settings.go

```diff
@@ -173,6 +174,7 @@ type logTrigger struct {
 	EventRateLimit           Setting[config.Rate]
 	FilterAddressLimit       Setting[int] `unit:"{address}"`
 	FilterTopicsPerSlotLimit Setting[int] `unit:"{topic}"`
+	EventSizeLimit           Setting[config.Size]
```

Comment:
Let's keep these alphabetical:
```suggestion
	EventSizeLimit           Setting[config.Size]
	FilterAddressLimit       Setting[int] `unit:"{address}"`
	FilterTopicsPerSlotLimit Setting[int] `unit:"{topic}"`
```